src/redux/selectors/moviesCount.ts

```diff
@@ -0,0 +1,4 @@
+import { RootState } from 'redux/store';
```

Comment:
Both `moviesCountSelector` and `selectedMovieSelector` are referring to `Movie` entity, you can merge these selectors into one file. File should just contain all items related to particular entity!